tests/call_run_on_script_with_keras_save_and_load.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
It would be useful to separate unit tests from integration tests (end-to-end workflow tests).

Reply:
This should be an integration test. For now i am adding them as scripts here as we do not yet know integration test framework for this repo.

Reply:
For now we can just have an `integration` subfolder in the `tests` folder, so as to easily identify integration tests.

Reply:
Sounds good, added a folder.